allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -157,7 +192,87 @@ def get_rolling_performance_df(self):
 
         return performance_metrics_df
 
-    def get_performance_metrics(self, engaged_trial_reward_rate_threshold=2):
+    def get_performance_metrics(
+            self,
+            engaged_trial_reward_rate_threshold: float = 2.0
+        ) -> dict:
+        """Get a dictionary containing a subject's behavior response
+        summary data.
+
+        Parameters
+        ----------
+        engaged_trial_reward_rate_threshold : float, optional
```

Comment:
Is the description that I have for this correct?

I also changed the type from int to float, since it's conceivable that one might have a fraction threshold of reward/minute before considering the subject 'engaged'?

Reply:
Yes, that looks correct to me.